kostyor/tests/unit/test_db_api.py

```diff
@@ -57,21 +57,40 @@ def test_get_all_clusters(self):
 
         expected = db_api.get_clusters()
 
-        self.assertEqual(len(expected['clusters']), 10)
+        self.assertEqual(len(expected['clusters']), 11)
```

Comment:
Maybe it would be better to check that the result list include10 these pre-created clusters test0...test9? Instead of checking list length, which is always greater than 0, because we create a single cluster in setUp method.